### PR TITLE
[ACR] Rename `ArtifactManifestProperties.Size`->`SizeInBytes`

### PR DESCRIPTION
**Pull request by Mohit-Chakraborty**

### CODE REVIEW

**Comment by Mohit-Chakraborty** [comment]
The one other potential rename we wanted to investigate was for `RegistryArtifact.GetTagPropertiesCollection()`.

From the feedback I got, we can use the `Collection` suffix as we have now. Another option is to use `All`, giving us `GetTagProperties()` and `GetAllTagProperties()` as the two variants.

So, the choice is between `GetTagPropertiesCollection()` and `GetAllTagProperties()` for the pageable version of the method.
I like what we have currently, want to get feedback to update in this PR.

**Review by AlexGhiondea — APPROVED** [review]

**Comment by annelo-msft** [comment]
> So, the choice is between GetTagPropertiesCollection() and GetAllTagProperties() for the pageable version of the method. I like what we have currently, want to get feedback to update in this PR.

Were you able to look into what others HLC libraries do when we have this plural situation?  I believe @KrzysztofCwalina's guidance was to be consistent with the existing precedent if there is one.  In LLC, we use the `All` to disambiguate.

**Comment by Mohit-Chakraborty** [comment]
> > So, the choice is between GetTagPropertiesCollection() and GetAllTagProperties() for the pageable version of the method. I like what we have currently, want to get feedback to update in this PR.
> 
> Were you able to look into what others HLC libraries do when we have this plural situation? I believe @KrzysztofCwalina's guidance was to be consistent with the existing precedent if there is one. In LLC, we use the `All` to disambiguate.

I didn't find any instance of this particular case. There are several places where we use the `Collection` suffix, not necessarily for this reason though. That is to clarify that the method returns a collection or an instance of a type that represents a collection.

**Comment by annelo-msft** [comment]
If we go with the Collection suffix over including `All`, we'll want to change LLC to adopt this convention as well.  Let's get @KrzysztofCwalina's perspective on this.

**Comment by annelo-msft** [comment]
Per @KrzysztofCwalina:

> I prefer the GetAll convention. The "Collections" suffix sounds like a Hungarian convention, i.e. refers to .NET type name as opposed to API semantics. 
> 
> Also, I think the "All" convention is used in some .NET APIs, but not the "Collections" convention.
> 
> Having said that the "All" convention does not work in all cases. For example, imagine an API that returns a collection of items/resources but takes a filter as a parameter. It might be misleading to call the method GetAll(string filter) given it won't return all items unless the filter is "*".
> 
> It would be good to think what we would use if we ran into such APIs with a filter.

**Comment by Mohit-Chakraborty** [comment]
@annelo-msft, @KrzysztofCwalina - I have updated the API names as per our discussion.

I will add an entry to [the guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#general-client-standardize-verbs) to the effect of -
GetAll<resource_name> when <resource_name> is itself in plural form, like `Properties`.

File: sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Generated/Models/ArtifactManifestProperties.cs [hunk 1]
```diff
@@ -72,8 +72,6 @@ internal ArtifactManifestProperties(string registryLoginServer, string repositor
         public string RepositoryName { get; }
         /// <summary> Manifest. </summary>
         public string Digest { get; }
-        /// <summary> Image size. </summary>
-        public long? Size { get; }
```
Comment (annelo-msft) [inline sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Generated/Models/ArtifactManifestProperties.cs:76]: nit: should this change to SizeInBytes, or did I just miss seeing that elsewhere?

**Review by annelo-msft — APPROVED** [review]
Cool, I see we're aligning with the LLC precedent.  Thanks for doing this!

File: sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Generated/Models/ArtifactManifestProperties.cs (hunk 1 above)
Comment (Mohit-Chakraborty) [reply sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Generated/Models/ArtifactManifestProperties.cs:76]: Yes, this has been renamed to `SizeInBytes`. So it has been renamed from here.

**Pull request merged**